src/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Wbpack Exercise</title>
+</head>
```

Comment:
- [Optional] I think it would be better to write a descriptive title related to the project you are working on. 
> Descriptive HTML page titles help users quickly understand a web page’s content.

for example 
```
<title> To Do List</title>
```